ui/selectors/selectors.js

```diff
@@ -2860,15 +2879,25 @@ export function getAllAccountsOnNetworkAreEmpty(state) {
 }
 
 export function getShouldShowSeedPhraseReminder(state) {
-  const { tokens, seedPhraseBackedUp, dismissSeedBackUpReminder } =
+  const { tokens, seedPhraseBackedUp, dismissSeedBackUpReminder, isUnlocked } =
     state.metamask;
 
+  const primaryKeyring = getMetaMaskKeyrings(state)[0];
```

Comment:
To future-proof this:

```suggestion
  const primaryKeyring = getMetaMaskHdKeyrings(state)[0];
```